src/inmanta/execute/scheduler.py

```diff
@@ -262,7 +262,7 @@ def define_types(self, compiler: "Compiler", statements: Sequence["Statement"],
 
         self.types = {k: v for k, v in types_and_impl.items() if isinstance(v, Type)}
 
-    def anchormap(
+    def anchormap_extended(
```

Comment:
We could go with just `get_anchormap`? Because this name only makes sense with respect to the old one, it's not really standalone.